dotnet/src/SemanticKernel.Abstractions/IKernel.cs

```diff
@@ -44,18 +43,6 @@ public interface IKernel
     [Obsolete("This will be removed in a future release. Use RegisterPlugin instead.")]
     ISKFunction RegisterCustomFunction(ISKFunction customFunction);
```

Comment:
I thought I deleted this.  Given all the breaking changes we're making, probably makes sense to delete all the Obsolete stuff that's in the repo. @markwallace-microsoft? That can obviously be a separate PR.

Reply:
Agreed, let's clean out obsolete stuff as we go